avocado/plugins/runner_nrunner.py

```diff
@@ -156,7 +156,11 @@ def _get_all_runtime_tasks(self, test_suite):
         for index, task in enumerate(test_suite.tests, start=1):
             task.known_runners = nrunner.RUNNERS_REGISTRY_PYTHON_CLASS
             # this is all rubbish data
-            test_id = TestID("{}-{}".format(test_suite.name, index),
+            if test_suite.name:
+                prefix = "{}-{}".format(test_suite.name, index)
+            else:
+                prefix = index
+            test_id = TestID(prefix,
```

Comment:
Possibly even simpler:
```python
test_id = TestID("{}-{}".format(test_suite.name, index) if test_suite.name else index,
```
or I would at least use the shorter `prefix` for both the nrunner and runner modules instead of `prefix` and `suite_prefix` as they essentially mean the same thing in both cases.

Reply:
I did the break because of the 80 columns, So, I'm think that is fine the current approach.